src/components/FeeSelector/shared.tsx

```diff
@@ -1,39 +1,39 @@
 import { Trans } from '@lingui/macro'
+import { ChainId, SUPPORTED_CHAINS } from '@uniswap/sdk-core'
 import { FeeAmount } from '@uniswap/v3-sdk'
-import { ALL_SUPPORTED_CHAIN_IDS, SupportedChainId } from 'constants/chains'
 import type { ReactNode } from 'react'
 
 export const FEE_AMOUNT_DETAIL: Record<
   FeeAmount,
-  { label: string; description: ReactNode; supportedChains: SupportedChainId[] }
+  { label: string; description: ReactNode; supportedChains: ChainId[] }
 > = {
   [FeeAmount.LOWEST]: {
     label: '0.01',
     description: <Trans>Best for very stable pairs.</Trans>,
     supportedChains: [
-      SupportedChainId.ARBITRUM_ONE,
-      SupportedChainId.BNB,
-      SupportedChainId.CELO,
-      SupportedChainId.CELO_ALFAJORES,
-      SupportedChainId.MAINNET,
-      SupportedChainId.OPTIMISM,
-      SupportedChainId.POLYGON,
-      SupportedChainId.POLYGON_MUMBAI,
+      ChainId.ARBITRUM_ONE,
+      ChainId.BNB,
+      ChainId.CELO,
+      ChainId.CELO_ALFAJORES,
+      ChainId.MAINNET,
+      ChainId.OPTIMISM,
+      ChainId.POLYGON,
+      ChainId.POLYGON_MUMBAI,
     ],
   },
   [FeeAmount.LOW]: {
     label: '0.05',
     description: <Trans>Best for stable pairs.</Trans>,
-    supportedChains: ALL_SUPPORTED_CHAIN_IDS,
+    supportedChains: SUPPORTED_CHAINS.concat(),
```

Comment:
we need a mutable copy of `SUPPORTED_CHAINS`

For more info see https://stackoverflow.com/questions/54188364/how-can-i-convert-a-readonlyarrayany-to-any

Reply:
nit: `[...SUPPORTED_CHAINS]` feels a little more intuitive to me